client/client.go

```diff
@@ -31,11 +31,106 @@ import (
 	"github.com/openpubkey/openpubkey/gq"
 	"github.com/openpubkey/openpubkey/pktoken"
 	"github.com/openpubkey/openpubkey/pktoken/clientinstance"
+	"github.com/openpubkey/openpubkey/util"
 )
 
 type OpkClient struct {
-	Op   OpenIdProvider
-	CosP *CosignerProvider
+	Op     OpenIdProvider
+	CosP   *CosignerProvider
```

Comment:
I think it makes the most sense to private these variables as well so that we can control creation in the `New` function

Reply:
+1 

My plan to do this is as follows:
1. Make cosP private since nothing external depends on it. Keep Op public for now.
2. Create Getters for Op and CosP. This will set us up for making OpkClient an interface.
3. Get PRs into downstream projects to move them to client.New() and OpkClient.Auth().
4. Once they are moved over, make op private.

I can do 1 and 2 in this PR and then 4 in a separate PR that makes OpkClient.OidcAuth() private as well

Reply:
Just checked in code that gets us 1 and 2. Once this PR is merged, I can create the PRs to switch everyone over to new.Client and  Auth(). None of the unittests or examples depend on cosP or Op being public values setting us up for 3.